cloudformation.json

```diff
@@ -81,7 +99,11 @@
                     "logs:GetLogEvents",
                     "logs:FilterLogEvents",
                     "cloudwatch:GetMetricData",
-                    "cloudwatch:GetMetricStatistics"
+                    "cloudwatch:GetMetricStatistics",
+                    "ec2:CreateNetworkInterface",
+                    "ec2:DescribeNetworkInterfaces",
+                    "ec2:CreateNetworkInterfacePermission",
+                    "ec2:DeleteNetworkInterface"
```

Comment:
These are required to deploy lambda functions into subnets

Reply:
I suppose that it would make sense to also wrap these up in the conditional logic :shrug:

Reply:
Yes. :) I found that the simplest way of doing this is to make the "PolicyDocument" depend on the VPC condition you created.